angular-ex-6/src/app/app.component.html

```diff
@@ -0,0 +1,93 @@
+<h1>Form Builder, Form Group, Form Control, Validators and ngsubmit</h1>
+
+<form class="form" [formGroup]="profileForm">
+  <div class="field">
+    <label class="label">username</label>
+    <input
+      id="name"
+      type="text"
+      class="input"
+      formControlName="name"
+      (click)="submit = false"
+      [class.invalid]="
+        profileForm.get('name')?.invalid &&
```

Comment:
Create a function that checks all these conditions and returns only a boolean to simplify reading the html code and format the file